js/main.js

```diff
@@ -26,8 +26,10 @@ function submitForm(event) {
       notes: $notes.value,
       entryId: data.editing
     });
-
-    entryList.prepend(newEntryCreation(data.editing));
+    updateData(data.editing);
+    var li = entryList.querySelectorAll('li');
+    li[data.editing] = newEntryCreation(data.editing);
+    loadPage();
```

Comment:
This is a bunch of work for the browser. Instead of destroying and recreating the elements, you should find the one that needs to be updated and replace just that one. Once you revise that, remove the part from `loadPage` on deleting existing `li` elements, since there should be no `li` elements when the page loads, and that is what the function is meant to be for.